docs/blog/2018-10-25-unstructured-data/index.md

```diff
@@ -105,21 +109,23 @@ The [`createPage` action](/docs/actions/#createPage) is passed an object contain
 
 In our example, we're accessing the context as [props to the component](https://github.com/jlengstorf/gatsby-with-unstructured-data/blob/0a91d87b9d4d24a0e6b04b33cc271e054b7467b6/src/templates/all-pokemon.js#L4). This allows us to completely circumvent Gatsby’s data layer; it’s just props.
 
-```jsx{1,3,5,12-14}:title=src/templates/all-pokemon.js
-export default ({ pageContext: { allPokemon } }) => (
+```jsx:title=src/template/all-pokemon.js
+export default ({ pageContext: { allPokemon } }) => ( // highlight-line
     {...}
-        {allPokemon.map(pokemon => (
+        {allPokemon.map(pokemon => ( // highlight-line
             <li
-                key={pokemon.id}
+                key={pokemon.id} // highlight-line
```

Comment:
Be careful of JSX here. We want these snippets to remain valid JSX, so it'll end up looking something more like this:

````
```jsx:title=src/templates/all-pokemon.js
export default ({ pageContext: { allPokemon } }) => ( // highlight-line
    {...}
        {allPokemon.map(pokemon => ( // highlight-line
            <li
                key={pokemon.id} // highlight-line
                style={{
                    textAlign: 'center',
                    listStyle: 'none',
                    display: 'inline-block'
                }}
            >
            {/* highlight-start */}
                <Link to={`/pokemon/${pokemon.name}`}>
                    <img src={pokemon.sprites.front_default} alt={pokemon.name} />
                    <p>{pokemon.name}</p>
            {/* highlight-end */}
                </Link>
            </li>
        ))}
    {...}
);
```
````